src/constants.ts

```diff
@@ -510,7 +508,7 @@ export const SUPPORTED_STU3_RESOURCES: STU3Resource[] = [
     'VisionPrescription',
 ];
 
-export const PATIENT_COMPARTMENT_RESOURCES: R4Resource[] = [
+export const R4_PATIENT_COMPARTMENT_RESOURCES: R4Resource[] = [

```

Comment:
Do compartments reside outside of exports/imports?

Reply:
Yep, compartments can exist outside of export/imports. 
https://www.hl7.org/fhir/compartmentdefinition.html
They can
```
* Function as an access mechanism for finding a set of related resources quickly
* Provide a definitional basis for applying access control to resources quickly

```